scripts/performance-tests-kevm.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+KEVM_VERSION='v1.0.335'
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+
+MASTER_COMMIT="$(git rev-parse origin/main)"
+
+FEATURE_BRANCH_NAME="$(git rev-parse --abbrev-ref HEAD)"
+FEATURE_BRANCH_NAME="${FEATURE_BRANCH_NAME//\//-}"
+
+if [[ $FEATURE_BRANCH_NAME == "master" ]]; then
+  FEATURE_BRANCH_NAME="feature"
+fi
+
+# Create a temporary directory and store its name in a variable.
+TEMPD=$(mktemp -d)
+
+# Exit if the temp directory wasn't created successfully.
+if [ ! -e "$TEMPD" ]; then
+    >&2 echo "Failed to create temp directory"
+    exit 1
+fi
+
+# Make sure the temp directory gets removed on script exit.
+trap "exit 1"           HUP INT PIPE QUIT TERM
+trap 'rm -rf "$TEMPD"'  EXIT
+
+
+feature_shell() {
+  GC_DONT_GC=1 nix develop . --extra-experimental-features 'nix-command flakes' --override-input k-framework/booster-backend $SCRIPT_DIR/../ --command bash -c "$1"
+}
+
+master_shell() {
+  GC_DONT_GC=1 nix develop . --extra-experimental-features 'nix-command flakes' --override-input k-framework/booster-backend github:runtimeverification/hs-backend-booster/$MASTER_COMMIT --command bash -c "$1"
+}
+
+cd $TEMPD
+git clone --depth 1 --branch $KEVM_VERSION https://github.com/runtimeverification/evm-semantics.git
+cd evm-semantics
+git submodule update --init --recursive --depth 1 kevm-pyk/src/kevm_pyk/kproj/plugin
+
+
+
+feature_shell "make poetry && poetry run -C kevm-pyk -- kevm-dist --verbose build plugin haskell --jobs 4"
+
+feature_shell "make test-prove-pyk PYTEST_PARALLEL=8 PYTEST_ARGS='--timeout 7200 -vv --use-booster' > $SCRIPT_DIR/kevm-$KEVM_VERSION-$FEATURE_BRANCH_NAME.log"
+
+if [ ! -e "$SCRIPT_DIR/kevm-$KEVM_VERSION-master-$MASTER_COMMIT.log" ]; then
+  master_shell "make test-prove-pyk PYTEST_PARALLEL=8 PYTEST_ARGS='--timeout 7200 -vv --use-booster' > $SCRIPT_DIR/kevm-$KEVM_VERSION-master-$MASTER_COMMIT.log"
+fi
+
+cd $SCRIPT_DIR
+grep ' call  ' kevm-$KEVM_VERSION-$FEATURE_BRANCH_NAME.log > kevm-$KEVM_VERSION-master-$MASTER_COMMIT.$FEATURE_BRANCH_NAME
+grep ' call  ' kevm-$KEVM_VERSION-master-$MASTER_COMMIT.log > kevm-$KEVM_VERSION-master-$MASTER_COMMIT.master
```

Comment:
Maybe we can use `sed` and cut out the `" call "` string to produce data which can also be fed to `gnuplot`?
`sed  -n -e 's/^\([0-9.]*\)s call  *.*\[\(..*\)\]$/\2 \1/p' <logfile>` produces lines with format `<test-name> <time-in-seconds>` (without the `src/.../test_bla` prefix).
We could also directly pass in the output of `grep` to the script, using `python3 compare.py <(grep ' call  ' ...) <(grep ' call  ' ...)`